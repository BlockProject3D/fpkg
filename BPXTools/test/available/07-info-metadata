export NAME="Info (Metadata)"
export COMMAND="-f test.bpx info -sm"
export DESC="Test the info command"
export STATUS=0