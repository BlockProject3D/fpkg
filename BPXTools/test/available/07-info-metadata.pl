$Test = {
    Name => "Info (Metadata)",
    Command => "-f test/available/test.bpx info -sm",
    Description => "Test the info command",
    Status => 0
};

sub TestBegin {

}

sub TestEnd {
    return 1;
}
