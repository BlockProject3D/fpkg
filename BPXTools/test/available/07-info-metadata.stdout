====> BPX Main Header <====
Type: P
Version: 1
File size: 1629
Number of sections: 2
====> End <====

====> BPX TypeExt <====
Architecture: Any
Platform: Any
Generator: PK
====> End <====

====> BPX Section Header Table <====
Section #0:
	Type: 255
	Size (after compression): 12
	Size: 12
	Flags:  CheckWeak
Section #1:
	Type: 1
	Size (after compression): 1529
	Size: 1529
	Flags:  CheckWeak
====> End <====

